game.js

```diff
@@ -366,9 +366,15 @@ class Game {
             return;
         if (this.state.phase !== 'play')
             return;
-        if( config.greeting && !this.greeted && this.state.moves.length < (2 + this.state.handicap) ){
-            this.sendChat( config.greeting, "discussion");
-            this.greeted = true;
+        if (!this.greeted && this.state.moves.length < (2 + this.state.handicap)) {
+            if (config.greeting) {
+                this.sendChat(config.greeting, "discussion");
+                this.greeted = true;
```

Comment:
outside of the if

Reply:
thanks, fixed in https://github.com/online-go/gtp2ogs/pull/274/commits/d57c8797786ea6da968473e0fe18cb71b2cf66e8